src/helpers/git_helpers/branch_helpers.sh

```diff
@@ -187,17 +190,24 @@ function remote_only_merged_branches {
 }
 
 
-function undo_steps_for_create_and_checkout_feature_branch {
-  local current_branch=$(get_current_branch_name)
+function undo_steps_for_create_and_checkout_branch {
   local branch_to_create="$1"
 
+  local current_branch=$(get_current_branch_name)
   echo "checkout $current_branch"
+
```

Comment:
I think this should follow a similar style as `undo_steps_for_create_and_checkout_feature_branch`.
All the variable definitions, then a empty line, then all the echos.